functions/AutoRemediateCFM-005.js

```diff
@@ -19,20 +18,21 @@ module.exports.handler = (event, context, callback) => {
     StackName: event.resource
   }
 
-  const CloudFormation = new AWS.CloudFormation({ region: event.region })
-
-  CloudFormation.updateTerminationProtection(params, function (err, result) {
-    if (err) {
-      console.log('Error', err)
-      return handleError(err.message ? err.message : 'Failed to enable CloudFormation Stack Termination Protection')
-    }
+  const CloudFormation = new CloudFormationClient({ region: event.region })
 
+  try {
+    const result = await CloudFormation.send(new UpdateTerminationProtectionCommand(params))
     console.log('Result', result)
     return callback(null, 'Successfully processed event')
```

Comment:
♻️ As this is now `async` there is no need to use the `callback` pattern

```suggestion
    return 'Successfully processed event';
```

Note: This comment applies to all other similar files